File: Problems & Solutions/1470. Shuffle the Array/06 February 2023/Approach 1 - Simple Iteration/Shuffle the Array.swift
class Solution {
    func shuffle(_ nums: [Int], _ n: Int) -> [Int] {
        var result = Array(repeating: 0, count: 2 * n)
        for i in 0..<n {
            result[2 * i] = nums[i]
            result[2 * i + 1] = nums[n + i]
        }
        return result
    }
}

File: Problems & Solutions/1470. Shuffle the Array/06 February 2023/Approach 1 - Simple Iteration/Shuffle the Array.js
var shuffle = function(nums, n) {
    let result = Array(2 * n);
    for (let i = 0; i < n; ++i) {
        result[2 * i] = nums[i];
        result[2 * i + 1] = nums[n + i];
    }
    return result;
};